.travis.yml

```diff
@@ -32,20 +32,31 @@ install:
   - pip install -r rootfs/dev_requirements.txt
   - make -C docs/ deps
   - GLIDE_HOME=/home/travis/.glide make -C client bootstrap
+  - make prep-bintray-json
```

Comment:
when I was testing out #71, I tried creating a central dir and upload it all to S3, so I used the following:

```
before_deploy: mkdir build_artifacts
```

What I found was the build failed during `before_deploy` because the directory already existed. That was because there was two deploy targets. As soon as I commented out one, the build succeeded.

TL;DR I found it out through trial and error